docs/integrations/prometheus.md

```diff
@@ -92,3 +92,11 @@ remote_read:
 remote_write:
   - url: "http://localhost:7201/api/v1/prom/remote/write"
 ```
+
+Also, if you want to see metrics generated by the coordinator/query process such as number of queries, etc., you should add the following job under `scrape_configs`:
```

Comment:
nit: replace "coordinator/query" with "m3coordinator or m3query" for consistency?